hedera-node/hedera-file-service-impl/src/main/resources/genesis/simpleFeesSchedules.json

```diff
@@ -83,7 +84,7 @@
         },
         {
           "name": "ConsensusUpdateTopic",
-          "baseFee": 3240000,
+          "baseFee": 1900000,
```

Comment:
Same here .. shouldn't the base fee be `2200000`? I am curious how we are getting to this value